.github/workflows/add-voting-info.yml

```diff
@@ -0,0 +1,26 @@
+name: Add voting info to new issue
+
+on:
+  issues:
+    types:
+      - opened
+
+jobs:
+  add-voting-info:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: Append voting info to issue body
+        uses: julien-deramond/update-issue-body@v1
+        with:
+          issue-number: ${{ github.event.issue.number }}
+          body: |
+            > [!IMPORTANT]
+            > You can vote for this issue using the 👍 reaction. More votes increase the issue's priority
+            > for developers.
```

Comment:
A linebreak before `for developers` might slipped in that make the result deviate from the screenshot.

<img width="610" alt="Screenshot 2023-09-25 at 22 43 40" src="https://github.com/vlang/v/assets/34311583/84ee1ba0-df44-47f9-a65d-db8e775c800f">

Wouldn't a `Note` instead of `Important` fit better? 

Important is described as `Crucial information necessary for users to succeed.`

While Note is described as `Highlights information that users should take into account, even when skimming.`

Reply:
Sorry about the trouble on the closed PR @ArtemkaKun. Just a small thing I'm noticing here. I'll leave you a ping in case you don't see it on the old PR and if something should be changed.